### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by NxHunter**
You asked.

I made.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
 # wolcen_better_ui
+
+This mod attempts to allow huge amounts of personalisation to the current Wolcen UI. 
+
+Planned features of this modification are:
+
+
+* Creating a custom layout of the existing elements
```
Comment (sebastienvercammen) [inline README.md]: Too much whitespaces (empty lines above this one).

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+* More sockets and where to find them
+* dreais' source and another modders seb
+* Interesting and fun to read ReadMe. 
+* Adding brand new custom UI elements created by other people, or even yourself!
+* More stupidity from Nx.
+
+With all of these amazing features, you'd be crazy to miss out on this wolcen_better_ui modification pack!
+
+So what are you waiting for? All this can be yours with a simple click of a button!
+Download now!
```
Comment (sebastienvercammen) [inline README.md]: Too little whitespace (needs an empty line above this one).

File: README.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+This mod attempts to allow huge amounts of personalisation to the current Wolcen UI. 
+
+Planned features of this modification are:
+
+
+* Creating a custom layout of the existing elements
+* Disabling exisiting UI elements
+* A 7 day weightloss program
+* More sockets and where to find them
+* dreais' source and another modders seb
```
Comment (sebastienvercammen) [inline README.md]: Capitalization of first name. 😌

File: README.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+* Interesting and fun to read ReadMe. 
+* Adding brand new custom UI elements created by other people, or even yourself!
+* More stupidity from Nx.
```
Comment (sebastienvercammen) [inline README.md]: > * More stupidity from Nx.

Best part of the PR. Well done. 👏

Inconsistent punctuation though: either add punctuation to all the bullet points, or don't use it for any of them.

File: README.md [hunk 5]
```diff
@@ -1 +1,20 @@
 # wolcen_better_ui
+
+This mod attempts to allow huge amounts of personalisation to the current Wolcen UI. 
+
+Planned features of this modification are:
+
+
+* Creating a custom layout of the existing elements
+* Disabling exisiting UI elements
+* A 7 day weightloss program
```
Comment (sebastienvercammen) [inline README.md]: > * A 7 day weightloss program

No.

Also, weight loss. *

File: README.md [hunk 6]
```diff
@@ -1 +1,20 @@
...
+* More sockets and where to find them
```
Comment (sebastienvercammen) [inline README.md]: [Fantastic Sockets and Where to Find Them](https://www.imdb.com/title/tt3183660/)

**Comment by dreais** [comment]
i'll wait for these modifications to be applied!

File: README.md (hunk 3 above)
Comment (dreais) [reply README.md]: (dreais not being my name, i'd leave my lowercase letter as it is)

**Comment by dreais** [comment]
i'll accept that PR now because im a lazy ass so feel free to re-submit the corrected one

**Pull request merged**